### PR TITLE
feat(core): [LW-9408] create All Set component

### PR DESCRIPTION
**Pull request by renanvalentin**
# Checklist

- [x] JIRA - [LW-9408](https://input-output.atlassian.net/browse/LW-9408)
- [x] Screenshots added.

---

## Screenshots

Screenshot 2024-01-08 at 09 00 11

capture-cbcfb047

[LW-9408]: https://input-output.atlassian.net/browse/LW-9408?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/ui/components/SharedWallet/AllSet/AllSet.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      advancedFeatures: Educational;
+      tips: Educational;
+    };
+    goToSharedWallet: string;
+  };
+}
+
+export const AllSet = ({ translations }: Props): JSX.Element => (
+  <Flex h="$fill" flexDirection="column">
+    <Box mb={'$24'}>
```
Comment (lucas-barros) [inline packages/core/src/ui/components/SharedWallet/AllSet/AllSet.tsx:25]: Why not use the EducationalCard.Root here?

**Review by pczeglik-iohk — APPROVED** [review]
Code looks good. Thank you @renanvalentin

**Comment by pczeglik-iohk** [comment]
@renanvalentin I have one open question. I see you have added a bunch of icons, but I'm missing (or I'm not able to find) a place in our storybook where all available icons are presented. Do we have such screen?

In Figma I see `Default icons` section which presents all icons in dark and light mode.

I would like to have one global screen with all available icons to be used :)

File: packages/core/src/ui/components/SharedWallet/AllSet/AllSet.tsx (hunk 1 above)
Comment (renanvalentin) [reply packages/core/src/ui/components/SharedWallet/AllSet/AllSet.tsx:25]: This screen only needs the EducationalCardItem, as the Root would behave like a listing.

**Comment by renanvalentin** [comment]
> @renanvalentin I have one open question. I see you have added a bunch of icons, but I'm missing (or I'm not able to find) a place in our storybook where all available icons are presented. Do we have such screen?
> 
> In Figma I see `Default icons` section which presents all icons in dark and light mode.
> 
> I would like to have one global screen with all available icons to be used :)

We do not have a page listing all the icons used yet. There has been an ongoing discussion of how to organize the icons.

**Review by lucas-barros — APPROVED** [review]

**Comment by DominikGuzei** [comment]
@renanvalentin i can see that one of the stories is not under the section "Shared Wallets" in Storybook, is that intentional?

File: packages/core/src/ui/components/SharedWallet/AllSet/AllSet.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { sx, Box, Flex, Text, Button, EducationalCard } from '@lace/ui';
+import { ReactComponent as BookGradient } from '@lace/ui/dist/assets/icons/book-gradient.component.svg';
```
Comment (DominikGuzei) [inline packages/core/src/ui/components/SharedWallet/AllSet/AllSet.tsx:3]: good idea to put all the icons in `@lace/ui`, will update my PR to do the same 👍

File: packages/ui/src/design-system/educational-card/educational-card-item.component.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import { Box } from '../box';
+import { Flex } from '../flex';
+import * as Text from '../typography';
+
+import * as cx from './educational-card-item.css';
+
+import type { OmitClassName } from '../../types';
+
+type Props = OmitClassName<'div'> & {
```
Comment (DominikGuzei) [inline packages/ui/src/design-system/educational-card/educational-card-item.component.tsx:12]: can you elaborate on omitting the `div` class name?

**Review by DominikGuzei — APPROVED** [review]
Left two comments, LGTM 🚢

File: packages/ui/src/design-system/educational-card/educational-card-item.component.tsx (hunk 3 above)
Comment (renanvalentin) [reply packages/ui/src/design-system/educational-card/educational-card-item.component.tsx:12]: @DominikGuzei these high-level components should be considered sealed and ready-to-use. Most of the UI toolkit components follow this approach.

**Pull request merged**